misk/src/main/kotlin/misk/security/csp/Csp.kt

```diff
@@ -0,0 +1,15 @@
+package misk.security.csp
+
+// This annotation allows misk endpoints to define their Content-Security-Policy directive.
+// See https://web.archive.org/web/20220906063156/https://content-security-policy.com/
+// for a reference that's up to date when this annotation was added.
+//
+// Production CSPs can be very long so a future contribution should add the ability to define
+// CSP into YAML configs and point the @Csp annotation to the config value.
+//
+// This annotation is currently dumb, it adds the rules passed to a Content-Security-Policy
+// header directive, with no inspection or validation. A future change could parse the policy.
+//
+// Developers using this should check out the various browser extensions to verify the actual
+// CSP on their webpages.
+annotation class Csp(val rules: Array<String>)
```

Comment:
I'd lean towards CSP being written out to reduce use of acronyms in code

Reply:
I can land a follow up to make it @ContentSecurityPolicy. Francisco, what do you think?

Reply:
Done, follow up PR coming.